lib/src/gantt_default_day_header.dart

```diff
@@ -3,36 +3,55 @@ import 'package:flutter/material.dart';
 import 'week_day.dart';
 
 class GanttChartDefaultDayHeader extends StatelessWidget {
+  static const defaultColor = Colors.black;
+  static const defaultBackgroundColor = Colors.white;
+  static const defaultHolidayColor = Colors.white;
+  static final defaultHolidayBackgroundColor = Colors.grey.shade800;
```

Comment:
what do you think about removing `const/final` so they can be changed later by the developers, e.g.:
`GanttChartDefaultDayHeader.defaultColor = SomeColor;`
might be useful when dealing with multiple gantt charts on the same app

Reply:
> what do you think about removing `const/final` so they can be changed later by the developers, e.g.: `GanttChartDefaultDayHeader.defaultColor = SomeColor;` might be useful when dealing with multiple gantt charts on the same app

Uhm, IMHO those are like "platform defaults" which should not be changed, they represent "standard" colors. Developers can still pass colors as arguments, so it should be flexible enough.

Reply:
ok then, I think PR should be fine now

Reply:
or do you have any additional changes in mind ?

Reply:
I think I'm done (for the time being) :smile: